kolibri/plugins/coach/assets/src/composables/useResourceSelection.js

```diff
@@ -67,6 +75,19 @@ export default function useResourceSelection() {
       }),
   });
 
+  const useSearchObject = useBaseSearch({
+    descendant: topic,
+  });
+  const searchFetch = {
+    data: useSearchObject.results,
+    loading: useSearchObject.searchLoading,
+    hasMore: computed(() => !!useSearchObject.more.value),
```

Comment:
Can this be reduced just to `hasMore: computed(() => useSearchObject.more.value)` ?

Reply:
Hi @ozer550! In this case its not the same, because `hasMore` should be a boolean value, and `useSearchObject.more.value` is an object, the "more" object. So to transform it to boolean we can do either `!!useSearchObject.more.value` or `Boolean(useSearchObject.more.value)`, but personally I like more the former :sweat_smile:.